### PR TITLE
solving issue #74

### PR DESCRIPTION
**Pull request by GautamV234**
## Description

Hello, Dr.@murphyk this PR latexifies and jaxifies two figures from chapter 17 of Book2.

### Figure Number

book2 - 17.3
book2 - 17.4 

### Figures

| Before PR | After PR |
| :-: | :-: |
|Fig_17 3_book|Fig_17 3

| Before PR | After PR |
| :-: | :-: |
|Fig_17 4_book|Fig_17 4

### Issue 
#739

## Steps

- [x] Changed the name of the variables/functions to PEP 8 style.
- [x] Added code to generate two figures as given in the book and Latexified them. 
- [x] Converted code to JAX.

## Checklist

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

## Potential problems/Important remarks

* Used TinyGP and its kernels to fit Gaussian Process.

### CODE REVIEW

File: notebooks/book2/17/gpKernelPlot.ipynb [hunk 1]
```diff
@@ -0,0 +1,369 @@
+{
```
Comment (patel-zeel) [inline notebooks/book2/17/gpKernelPlot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    def gpKernelPlot(seed):
</pre><p>Use PEP8 names for methods. <code>gp_kernel_plot</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (patel-zeel) [inline notebooks/book2/17/gpKernelPlot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #13.      periodic_kernel = kernels.ExpSineSquared(scale = 1.0,gamma=jnp.array(2.0))
</pre><p>This looks too streched. Maybe increase the period so that it looks good comparable to matern plots.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (patel-zeel) [inline notebooks/book2/17/gpKernelPlot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #14.      cosine_kernel = kernels.Cosine(scale=1.0)
</pre><p>Same comment as periodic kernel.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (patel-zeel) [inline notebooks/book2/17/gpKernelPlot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #29.        plt.title(kernel_name, fontsize=12)
</pre><p>Remove this from title. We will add it via latex.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by patel-zeel** [comment]
Remove titles from all figures.

**Comment by patel-zeel** [comment]
@GautamV234 We wanted to merge 17.3 and 17.4 to create a single figure with 16 different kernels in it (4x4 grid). Please collaborate with Aadesh to do this. Something like the following in terms of variety of kernels:

image

File: notebooks/book2/17/gpKernelPlot.ipynb (hunk 1 above)
Comment (GautamV234) [reply notebooks/book2/17/gpKernelPlot.ipynb:1]: image
Is this any better?

[the remark this answers is not in the document]
Comment (GautamV234) [reply notebooks/book2/17/gpKernelPlot.ipynb:1]: Done! Thanks

Comment (GautamV234) [reply notebooks/book2/17/gpKernelPlot.ipynb:1]: Done
